.github/workflows/integration.yml

```diff
@@ -104,6 +104,7 @@ jobs:
           source ~/.poetry/env
           poetry run coverage report
           poetry run coverage xml -o coverage-reports/coverage-report.xml
+          sed -i 's/filename="app/filename="api\/app/' coverage-reports/coverage-report.xml
```

Comment:
what does this do??

Reply:
Replace occurrences of 'filename="app' with 'filename="api/app" in coverage-report.xml - you want I should add a comment?

Reply:
Oh I see, Yeah that would be nice